django_hatstall/identities/templates/profiles/profile.html

```diff
@@ -66,7 +69,257 @@ <h1>Profile info / <small>{{profile.profile.uuid}}</small></h1>
                     </div>
                     <div class="form-group row">
                         <label for="profileCountry" class="col-sm-2 col-form-label"><b>Country</b></label>
-                        <input name="country" type="text" readonly class="col-sm-10 form-control-plaintext" id="profileEmail" value="{{profile.profile.country}}">
+                        <select id="countrySelect" name="country" type="text" disabled="true" class="col-sm-10 form-control" id="profileCountry" value="{{profile.profile.country.code}}">
+                          <option value="AF">Afghanistan</option>
```

Comment:
This information is already in the table countries (it was missing from the sample database, but now it is fixed) inside the SortingHat database. So we just need to collect it and to pass it ti the Django template, no need to have it hardcoded here.